aten/src/ATen/native/cuda/UpSampleNearest2d.cu

```diff
@@ -11,110 +11,114 @@ namespace at {
 namespace native {
 namespace {
 
+#define MAX_THREADS 512
+
+static int lastPow2(unsigned int n) {
```

Comment:
Why am I seeing this in so many places (last one I suspect was Thor's softmax)? Can it go into some helper header?